### ISSUE TITLE
iOS 13

### ISSUE THREAD

**Issue by iRayanKhan** [evandcoleman/homebridge-bond#21]
Hello everyone,

So iOS 13 is now in the hands of developers and there's been some changed to the Home App. 

Fans now group accessories with them, so the Fan and Light Accessory and now 1 (Still 2 accessories just show as 1 under the same icon). Now it also added the Fan Reset switch in it so you see the Fan Reset and Fan Light together. I was wondering if anyone could make a PR or a branch based off #19 to be able to have the "Fan Reset" switch not pair with the fan.

If this was confusing here's a tldr:
This plugin adds 3 accessories per fan (Fan, Light, and Fan Reset). iOS 13 now lets you tap on the fan accessory to control the Fan and Light in the same field. Since this plugin adds 3 accessories, it groups all 3 since they are linked (individual controls still work). I was just asking if there can be a way to have the "Fan Reset" accessory not group together with the new fan in iOS 13. 

<img width="269" alt="Screen Shot 2019-06-11 at 3 15 52 AM" src="https://user-images.githubusercontent.com/34495712/59255307-7ce3d600-8bf7-11e9-91ce-c4b0bd975498.png">

So then it should just look like the Fan and just a Light on the side and have the fan reset be separate. 
Paging @plainsane

**Comment by plainsane** [comment]
ill take a look at some point but im wrapped up at the moment.

**Comment by vdotn** [comment]
I don’t currently have a device I’m willing to sacrifice to beta1, but I’ll try to take a look at it once the public beta is released.

**Comment by iRayanKhan** [comment]
@vdotn I made a fork that removes the Fan Reset device, which we still need, but I'm not experienced enough to separate the accessory from being merged.

**Comment by aarons22** [comment]
I posted this in a couple other issues on this repo, but I just finished writing the initial version of the plugin using the v2 API: https://github.com/aarons22/homebridge-bond-v2

In doing so, I ended up dropping the Reset switch all together since the API exposes the current state of the device from the Bond. When HomeKit requests the current value (either on startup or when it's `Updating...`, I'm now querying the API to get the state. Now the Bond device is the source of truth, and I'm not really seeing a need for the Reset switch at this point!
